autosklearn/evaluation/train_evaluator.py

```diff
@@ -767,8 +716,12 @@ def partial_fit_predict_and_loss(self, fold: int, iterative: bool = False) -> No
                 test_indices=test_split,
                 add_model_to_self=True,
             )
-            train_loss = self._loss(self.Y_actual_train, train_pred)
-            loss = self._loss(self.Y_targets[fold], opt_pred)
+
+            # This is my best guess at what the X_data for these should be
```

Comment:
I'm not 100% sure, but do we need this code actually? I have the impression that the function `partial_fit_and_predict_iterative` does all the work that's done here anyways? Or maybe we should move this under the `else` block? As of now I'm also not 100% sure of what variables should be there...

Reply:
I would leave that up to you, I need to read and understand the whole evaluation module to make an informed decision about it.

Reply:
Then I guess we can leave this as it is, but I think we don't need `X_fold` here and can use `self.X_targets[fold]` instead?

Reply:
I think this was the issue causing test failures!